ui/app/AppLayouts/Browser/BrowserLayout.qml

```diff
@@ -274,7 +274,7 @@ StatusSectionLayout {
             anchors.left: parent.left
             anchors.right: parent.right
             z: 50
-            tabComponent: webEngineView
+            tabComponent: !!webEngineView ? webEngineView : null
```

Comment:
This component is defined below and should always be available.. When is this happening?

Reply:
when the browser view is loading.